src/Model/Factory/RencontreDetailsFactory.php

```diff
@@ -139,9 +141,11 @@ private function formatJoueurs(array $data, string $playerClubId): array
 
         $joueurs = [];
         foreach ($data as $joueurData) {
-            $nomPrenom = $joueurData[0];
-            [$nom, $prenom] = $this->nomPrenomExtractor->extractNomPrenom($nomPrenom);
-            $joueurs[$this->nomPrenomExtractor->removeSeparatorsDuplication($nomPrenom)] = $this->formatJoueur($prenom, $nom, $joueurData[1], $joueursClub);
+            if ($joueurData['nom'] && $joueurData['points']) {
```

Comment:
So in this way, we can have 4 players in left, and 3 players in right, is not a problem ?

Historically I wanted always same number of players on left and right